docs/reference/supervisor.md

```diff
@@ -1358,7 +1372,16 @@ wb_supervisor_node_reset_physics(node)
 
 ##### Description
 
-*stops the inertia of the given solid*
+*resets state of the given solid*
+
+The `wb_supervisor_node_save_state` function saves the state of the given node and all its descendant nodes.
+The state is saved internally under a key equal to the `state_name` parameter.
+Therefore, the state can be reverted using the `wb_supervisor_node_reset_state` function afterward.
+
+The `wb_supervisor_node_reset_state` function loads the state of the given node and all its descendant nodes.
+The function parameter `state_name` has to be a valid value, or the controller will fail.
+The valid `state_name` value means that the state is saved with the `wb_supervisor_node_reset_state` function, or a special value `__init__`.
+If the `node` parameter is equal to the root node and the `state_name` parameter is equal to `__init__` then the function will have a similar effect to the `wb_supervisor_simulation_reset` function, except it will not affect the user interface (for example, it will not reset the timer).
```

Comment:
We should explain what is a state. Does it include translation, rotation, inertia, appearance, etc?